lib/mayaUsd/ufe/UsdUndoAddNewPrimCommand.cpp

```diff
@@ -48,7 +48,7 @@ UsdUndoAddNewPrimCommand::UsdUndoAddNewPrimCommand(const UsdSceneItem::Ptr& usdS
     _stage = usdSceneItem->prim().GetStage();
     if (_stage) {
         // Append the parent path and the requested name into a full ufe path.
-        _newUfePath = appendToPath(ufePath, name);
+        _newUfePath = appendToPath(ufePath, name + std::to_string(1));

```

Comment:
Could just be name + "1".

Reply:
@ppt-adsk please see 324bff09d53cab1f693fed35c9f0a293b435a34c